types/firestore.d.ts

```diff
@@ -1964,13 +1980,16 @@ declare namespace FirebaseFirestore {
    * Firestore v1beta1 RPCs.
    * @deprecated Use v1 instead.
    */
-  export const v1beta1: any;
+  export const v1beta1: typeof import('./v1beta1/firestore_client');
```

Comment:
You can just do a regular import:

import v1beta1 from '...';
import {FirestoreClient, FirestoreAdminClient} from '...'

Reply:
> Approach LGTM. Some improvement suggestions.
> 
> I also wonder whether this should be a `fix` or even a `feat` since this is something that we want to create a release for,

'fix' should cause a patch release.

Reply:
I don't think top level imports in `.d.ts` files are allowed (see [stackoverflow](https://stackoverflow.com/questions/39040108/import-class-in-definition-file-d-ts)). I played around with it, but it always ended in sadness.

Reply:
Update: Following our conversation, changed to use `import` directly inline, rather than use `typeof`.